README.md

```diff
@@ -129,6 +129,13 @@ Both limits are defined within the _laser_1_ frame as shown in the image below.
 <img src="img/angle_limits.png" width="800px" alt="Limit visualization" title="Limit visualization">
 </p>
 
+### Additional Information
+The timestamps of the scan data published are computed to be close to reality and processing speed of incoming data was optimized to reduce the time for the scan data to be published. This should suffice for localization and slam algorithms. If you application benefits from getting the scan data any sooner there are two possiblities:
+
+* Use fragmented scans. This will publish the individual frames received by the PsenScan hardware imidiatly instead of waiting for a scan round to be completed and combined to a single laser scan message.
```

Comment:
```suggestion
* Use fragmented scans. This will publish the individual frames received by the PsenScan hardware immediately instead of waiting for a scan round to be completed and combined to a single laser scan message.
```